src/sentry/utils/sdk.py

```diff
@@ -458,6 +459,12 @@ def flush(
         "schedule-digests",
     ]
 
+    # Check if SENTRY_SPOTLIGHT is enabled and set additional environment variables to inject
+    # Spotlight Django middleware
+    if os.environ.get("SENTRY_SPOTLIGHT") == "1":
+        os.environ["SENTRY_SPOTLIGHT_ON_ERROR"] = "1"
+        os.environ["SENTRY_SPOTLIGHT_MIDDLEWARE"] = "1"
```

Comment:
These should default to `1` already, are they not?

Reply:
@BYK nah. They're not set. It doesn't inject the django middleware without it: https://github.com/getsentry/sentry-python/blob/a97c53ca697c1fd3132e5b3d5e67887d63187963/sentry_sdk/spotlight.py#L62-L172

Reply:
If these are not set, they default to `"1"` though: https://github.com/getsentry/sentry-python/blob/a97c53ca697c1fd3132e5b3d5e67887d63187963/sentry_sdk/spotlight.py#L219-L220

Reply:
@BYK oh right. the issue was the stale sidecar package; not these env variables not being set.